ingestion/src/metadata/ingestion/source/database/athena.py

```diff
@@ -21,12 +26,73 @@
     Source as WorkflowSource,
 )
 from metadata.ingestion.api.source import InvalidSourceException
+from metadata.ingestion.source import sqa_types
 from metadata.ingestion.source.database.common_db_source import CommonDbSourceService
 from metadata.utils.logger import ingestion_logger
 
 logger = ingestion_logger()
 
 
+def _get_column_type(self, type_):
```

Comment:
this is a copy-paste implementation of the athena dialect. having mixed feelings about rewritting